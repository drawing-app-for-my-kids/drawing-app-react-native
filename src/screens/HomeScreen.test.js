import "react-native";
import React from "react";
import {
  render,
  screen,
  fireEvent,
  waitFor,
} from "@testing-library/react-native";
// Note: test renderer must be required after react-native.
import renderer from "react-test-renderer";

import HomeScreen from "../screens/HomeScreen";

let component;
let testingLib;

describe("Testing HomeScreen", () => {
  beforeEach(() => {
    component = <HomeScreen />;
    testingLib = render(component);
  });

  it("정상적으로 렌더링이 되어야 합니다.", async () => {
    const rendered = await waitFor(() => renderer.create(component).toJSON());
    expect(rendered).toMatchSnapshot();
  });
});
