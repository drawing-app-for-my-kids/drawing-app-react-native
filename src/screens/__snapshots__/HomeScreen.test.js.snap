// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing HomeScreen renders without crashing 1`] = `null`;

exports[`Testing HomeScreen 정상적으로 렌더링이 되어야 합니다. 1`] = `null`;
